golang/Solutions/assgn1.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var file string = "history.txt"
+
+func check(e error) {
+	if e != nil {
+		os.Truncate(file, 0)
+		panic(e)
```

Comment:
Not a good practice to always panic. Instead, return the error and continue. Think of it like how our  current shell works